src/main/java/com/hrznstudio/galacticraft/screen/PlayerInventoryGCScreenHandler.java

```diff
@@ -177,12 +177,27 @@ public ItemStack transferSlot(PlayerEntity player, int index) {
         if (slotFrom != null && slotFrom.hasStack()) {
             ItemStack stackFrom = slotFrom.getStack();
             stack = stackFrom.copy();
-            if (index >= 0 && index < 8) {
-                if (!this.insertItem(stackFrom, 9, 48, true)) {
+
+            // Index of Indexes :)
+            // 0-3 (4): GC, armor slots;
+            // 4: GC, mask slot;
+            // 5: GC, oxygen gear;
+            // 6-7 (2): GC, oxygen tanks slots;
+            // 8-11 (4): GC, slots without any required item;
+            // 12-38 (27): MC, non-hotbar inventory slots;
+            // 39-48 (9): MC, hotbar slots.
+            if (index >= 0 && index < 12) {
```

Comment:
Index can't be less than zero, as line 176 would throw an error.
```suggestion
            if (index < 12) {
```